source/en/api/api_bandit.rst

```diff
@@ -55,6 +57,19 @@ We show each field below:
 
         * range of value: 0 <= ``seed`` <= :math:`2^{32} - 1`
 
+   epsilon_decreasing
+    :decreasing_rate:
+        Decreasing parameter of epsilon. The bigger this parameter becomes, the more slowly epsilon dcreases. (Float)
+
+        * Range: 0 < ``decreasing_rate`` < 1
+    
+    :seed(optional):
+        Specify random seed when you use ``random`` for unlearning strategy.
```

Comment:
This parameter is not for unlearning.
The same mistakes are found in other methods.